android/src/main/kotlin/io/customer/customer_io/extension/MapExt.kt

```diff
@@ -0,0 +1,36 @@
+package io.customer.customer_io.extension
+
+import io.customer.sdk.CustomerIOShared
+
+@Throws(IllegalArgumentException::class)
+internal inline fun <reified T> Map<String, Any?>.getPropertyUnsafe(key: String): T {
```

Comment:
```suggestion
internal inline fun <reified T> Map<String, Any?>.getValueUnsafe(key: String): T {
```

IMO, `value` reads easier since maps contain key value pairs.